en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
-**I want to build a drone with PX4 from scratch:**
+Both kits and complete vehicles usually include everything you need except for a battery and RC System.
+Kits are usually not hard to build, provide a good introduction to how drones fit together, and are relatively inexpensive.
+We provide generic instructions for assembly, such as [Assembling a Multicopter](../assembly/assembly_mc.md), and most kits come with specific instructions too.
 
-:::tip
+If the kits and complete drones aren't quite right then you can build one from scratch, but this requires more knowledge.
+[Airframe Builds](airframes/index.md) lists the supported frame starting points to give you some idea of what is possible.
 
-The "supported" vehicles are listed in the [Airframes Reference](airframes/airframe_reference.md).
-These are vehicles that have tested and tuned configurations that you can download using _QGroundControl_.
-
-:::
+### I have an RTF vehicle
 
-If you want to build a vehicle from scratch:
+Once you have purchased or built a Ready To Fly (RTF) vehicle that supports PX4 you will need to configure it and calibrate the sensors.
+Each vehicle type has its own configuration section that explains the main steps, such as [Multicopter Configuration/Tuning](config_mc/index.md).
 
-- Choose a frame - [Airframe Builds](airframes/index.md) lists the supported frames and provides detailed instructions for how to construct a subset of vehicles.
-- Choose a flight controller - see [Getting Started > Flight Controllers](getting_started/flight_controller_selection.md) and [Autopilot Hardware](flight_controller/index.md).
-- [Assembly](assembly/index.md) explains how to wire up the important peripherals to your autopilot.
-- [Basic Configuration](config/index.md) shows how to update your firmware and configure it with settings appropriate for your airframe.
-  This section also explains how to calibrate the main sensors (compass, gyro/IMU, airspeed etc.), and setup your remote control and safety features.
+Before you fly you should read [Operations](config/operations.md) to understand how to set up the safety features of your vehicle and the common behaviours of all frame types.
+Once you've done that you're ready to fly.
 
-Once you are ready to fly your vehicle, visit the [Flying](flying/index.md) section.
+Basic instructions for flying each vehicle type are provided in the respective sections, such as [Basic Flying (Multicopter)](flying/basic_flying_mc.md).
 
-**I want to add payload or a camera:**
+### I want to add payload or a camera
 
-The payloads section describes how to add a camera or how to configure PX4 to enable you to deliver packages.
+The [Payloads](payloads/index.md) section describes how to add a camera or how to configure PX4 to enable you to deliver packages.
 
-- [Payloads](payloads/index.md) describes how to integrate payloads
+After adding a payload you should retune the vehicle, which for multicopters you can do with [Autotune](config/autotune.md).

```

Comment:
```suggestion
After adding a heavy payload that's mounted far away from the center of gravity you might need to retune the vehicle, which for multicopters you can do with [Autotune](config/autotune.md).
```

Reply:
Thank you. That's the kind of thing I wasn't sure about.